easybuild/easyconfigs/p/Pysam/Pysam-0.10.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,51 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# 0.9.1.4:
+# Modified by: Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# @ adelaide.edu.au/phoenix
+
+easyblock = 'Bundle'
+
+name = 'Pysam'
+version = '0.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pysam-developers/pysam'
+description = """Pysam is a python module for reading and manipulating Samfiles. 
+ It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+parallel = 1
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('Cython', '0.25.2', versionsuffix),
+    ('ncurses', '6.0'),
+    ('zlib', '1.2.8'),  
+    ('cURL', '7.49.1'),  
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('pysam', version, {
```

Comment:
@robqiao Why is this being installed as a bundle when there's only one extension?

It should just be installed via `PythonPackage` directly, like in https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Pysam/Pysam-0.10.0-intel-2016b-Python-2.7.12.eb?

It makes sense to do it as a bundle if you include Cython as an extension in the Pysam installation, but that's not done here?